### PR TITLE
Added tests to ensure unsupported enums in modifyPermissions throws an error

### PR DESCRIPTION
**Pull request by sam-i-am012**
Addresses #80

### CODE REVIEW

**Review by korydraughn — COMMENTED** [review]
The test is good, but it needs to be in a new file since it's shared by two endpoints.

Add a file called `ModifyPermissionsOperationsTest.java` and put the test there.

**Comment by korydraughn** [comment]
Did something change?

I see a force-push, but the code looks unchanged.

**Comment by sam-i-am012** [comment]
No nothing has changed yet, I had just rebased onto main.

**Comment by sam-i-am012** [comment]
Ok, I removed the test from each individual endpoint and instead created a shared test file.

File: src/test/java/org/irods/Operations/DataObjectOperationsTest.java [hunk 1]
```diff
@@ -695,7 +695,6 @@ public void testModifyingPermissionsAtomically() {
             client.dataObject().remove(rodsToken, dataObject, 0, params);
         }
     }
-
```
Comment (korydraughn) [inline src/test/java/org/irods/Operations/DataObjectOperationsTest.java:698]: Please restore this line.

**Review by korydraughn — COMMENTED** [review]
Looks good. Just one thing to take care of and this will be ready.

File: src/test/java/org/irods/Operations/DataObjectOperationsTest.java (hunk 1 above)
Comment (sam-i-am012) [reply src/test/java/org/irods/Operations/DataObjectOperationsTest.java:698]: Done.

**Comment by sam-i-am012** [comment]
All existing review comments have been resolved.

**Comment by korydraughn** [comment]
Squash to taste. No pounds.

**Comment by sam-i-am012** [comment]
Done.

**Review by korydraughn — APPROVED** [review]
Pound it.

**Comment by sam-i-am012** [comment]
Done.

**Comment by korydraughn** [comment]
Merging.

**Pull request merged**